cmd/kpod/bud.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+
+var (
+	budFlags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "build-arg",
+			Usage: "`argument=value` to supply to the builder",
+		},
+		cli.StringSliceFlag{
+			Name:  "file, f",
+			Usage: "`pathname or URL` of a Dockerfile",
+		},
+		cli.StringFlag{
+			Name:  "format",
+			Usage: "`format` of the built image's manifest and metadata",
+		},
+		cli.BoolTFlag{
+			Name:  "pull",
```

Comment:
We should only expose docker build options.  Not these.

Reply:
OK, I'll keep an eyeball on the docker build params.  For this first pass I'll be sure to match all of those that buildah has.  If buildah doesn't have an equivalent, which I don't think is the case, I'll circle back and will update them in a separate PR.